8-lambda/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM amazonlinux:2017.03.1.20170812
+
+ENV NODE_VERSION 8.10.0
+ENV SERVICE_ROOT /service
+ENV SERVICE_USER service
+ENV YARN_VERSION 1.6.0
+
+RUN curl --silent --location https://rpm.nodesource.com/setup_8.x | bash -
+RUN curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo
+
+RUN yum install -y \
```

Comment:
Looks like you're installing node twice here. Once with the `+RUN curl --silent --location https://rpm.nodesource.com/setup_8.x | bash -` and here through `yum`. You might run into a unexpected version in the image.

Reply:
This is per the [node install instructions for CentOS](https://nodejs.org/en/download/package-manager/#enterprise-linux-and-fedora).  The `curl bash` add the repository, and the `yum install` does the actual install of a specific version.